### ISSUE TITLE
Episode Search: search box closes when clicking on an empty space

### ISSUE THREAD

**Issue by joashrajin** [Automattic/pocket-casts-android#861]
### Description

When you search for something like "AI" which provides no episode results, clicking on the grey area where the episodes would normally be showed causes the search box to close. 

Also, if we could get results for things like AI to show up, that would be quite nice :)

### Step-by-step reproduction instructions

1. Go to Discover
2. Search for "AI"
3. Click on the area at the bottom where you would normally see episode results

### Screenshots or screen recording

https://user-images.githubusercontent.com/16253818/229510211-ce13d9ff-3dd5-44a7-a810-82bec58f7586.mp4

### Did you search for existing bug reports?

- [X] I have searched for existing bug reports.

### Device, Operating system, and Pocket Casts app version

Samsung S21 FE, Android 13, Pocket Casts v7.35rc-2

**Comment by joashrajin** [comment]
![IMG_3927](https://user-images.githubusercontent.com/16253818/229547606-d8e43951-dce6-478d-b439-f1bd8ff9f8ce.png)

iOS says no episodes are available and doesn’t close the search box

**Comment by CookieyedCodes** [comment]
I personally don't mind this behavior, because it's a quick way to get back to discover 🤔 😉 (I do hate that I have to reach my hand for certain aspects of the app 😅)

**Comment by joashrajin** [comment]
@CookieyedCodes good point, but I feel it’s too abrupt. At least this should be possible if you swipe up or something, but not just clicking on the empty space.

**Comment by CookieyedCodes** [comment]
Fair enough, it is half the reason I suggest that hitting the discover icon would toggle between the search page (history) & discover(reload it when re-entering the page) it's the only thing I like about the Spotify expirence 😂
